### DISCUSSION TITLE
encode sharing link

### DISCUSSION THREAD

**Discussion by chraebsli** [community/community#56766] [Mobile]
### Select Topic Area

Bug

### Body

If I copy the link of a line in GitHub Mobile and paste it in a browser, the branch name (`#1-initialize-website`) is not encoded ⇒ I had to manually encode the `#` to `%23` in the branch name that the link works like expected.

**OS:**  
Android 13

**App Version:**  
1.115.0

**Comment by HenSquared** [comment]
Hi @chraebsli ,

I'll open an issue internally for our team to address this. Thanks for the feedback!

**Reply by HenSquared** [reply]
Do you also mind adding a recording or screenshot of where you are copying the branch URL from? thanks!

**Reply by chraebsli** [reply]
1. In GitHub Mobile, I clicked on any repository and changed the branch name (with a `#` in it). 
2. Then, I visited any file on this Branch and selected a line
3. When I clicked on the share button, I copied the link 
4. <img src="https://github.com/community/community/assets/73113302/fae99f47-9e16-4c26-b6ad-5e37a9add54f" width="200" />
5. I opened up a browser where I was logged in to GitHub and pasted the link. Then an error 404 occured.
6. <img src="https://github.com/community/community/assets/73113302/c0eddd30-9374-4269-ba68-6facd924a120" width="200" />
7. When I changed the `#` in the link to the encoded value `%23`, it worked
8. <img src="https://github.com/community/community/assets/73113302/1626894b-59de-4879-b023-1b5e112a5366" width="200" />

**Note:**  
Eventually, this error may occur with other special characters in:
* Branch name
* File name/ path

**Comment by ManishParashar0** [comment]
i dont know

**Comment by alcere** [comment]
Hi @chraebsli! We have solved the issue, and today's release (1.141.0) will contain the fix. Thanks for reporting!
